packages/trader/src/AppV2/Components/PurchaseButton/purchase-button.tsx

```diff
@@ -211,8 +211,9 @@ const PurchaseButton = observer(() => {
                         }
                         fullWidth
                         isOpaque
+                        isLoading={active_accu_contract?.is_sell_requested}
                         className='purchase-button purchase-button--single'
-                        disabled={is_accu_sell_disabled}
+                        disabled={!is_valid_to_sell}
```

Comment:
If `is_accu_sell_disabled` was replaced with `!is_valid_to_sell` here, do we need to do it also in ternary for `label` prop? Just asking

Reply:
I think should be okay , 
As if sell is requested we should only Close and no currency . as currency would keep on changing